pyntc/devices/nxos_device.py

```diff
@@ -294,17 +316,17 @@ def install_os(self, image_name, **vendor_specifics):
         self.native.show("terminal dont-ask")
         timeout = vendor_specifics.get("timeout", 3600)
         if not self._image_booted(image_name):
+            log.info("Host %s: Setting Image %s in boot options.", self.host, image_name)
             self.set_boot_options(image_name, **vendor_specifics)
+            log.info("Host %s: Waiting for device reload.", self.host)
             self._wait_for_device_reboot(timeout=timeout)
             if not self._image_booted(image_name):
                 log.error("Host %s: OS install error for image %s", self.host, image_name)
-                raise OSInstallError(hostname=self.facts.get("hostname"), desired_boot=image_name)
-            self.save()
-
```

Comment:
Save was removed, because we don't set boot options which should be saved like in cisco_ios. It was causing API exception:
`INSTALL OS failed: The command "copy run startup-config" gave the error "This command is not supported at this time. Try again when configuration restore is complete.`
After reload device is just loading up configuration.